css/properties/z-index.json

```
@@ -40,6 +40,38 @@
             "deprecated": false
           }
         },
+        "auto": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
```

It was in the first version of Edge.

```suggestion
              "edge": {
                "version_added": "12"
              },
```